src/components/FlatListItem/index.js

```diff
@@ -41,38 +44,38 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     paddingVertical: 16,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.alto,
+    borderBottomColor: colors.alto,
     paddingHorizontal: 20,
-    alignItems: 'center'
+    alignItems: 'center',
   },
   label: {
     fontFamily: fonts.inter[400],
     fontSize: 14,
     fontWeight: '400',
-    color: COLORS.black
+    color: colors.black,
   },
   desc: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     fontWeight: '400',
-    color: COLORS.gray8,
-    flexWrap: 'wrap'
+    color: colors.gray,
+    flexWrap: 'wrap',
   },
   labelIcon: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
     fontWeight: '500',
-    color: COLORS.black
+    color: colors.black,
   },
   contentText: {
     flexDirection: 'row',
     alignItems: 'center',
     flexWrap: 'nowrap',
-    width: 'auto'
+    width: 'auto',
   },
   containerLabel: {
     paddingLeft: 12,
     flexWrap: 'nowrap',
-    width: '88%'
-  }
+    width: '88%',
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This hunk seems to be primarily about updating the color values used in the styles. As with the previous hunk, this could potentially introduce issues if the colors are supposed to adapt to certain conditions. If the new `colors` module doesn't handle these cases, it might be worth considering reintroducing dynamic color handling.

```diff
- borderBottomColor: COLORS.alto,
- color: COLORS.black
- color: COLORS.gray8,
- color: COLORS.black
+ borderBottomColor: colors.alto,
+ color: colors.black,
+ color: colors.gray,
+ color: colors.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->